imports/plugins/included/taxes-avalara/server/methods/taxCalc.js

```diff
@@ -180,6 +200,18 @@ function avaGet(requestUrl, options = {}, testCredentials = true) {
 function avaPost(requestUrl, options) {
   const logObject = {};
   const pkgData = taxCalc.getPackageData();
+  // If package is not configured don't bother making an API call
+  if (!checkConfiguration(pkgData)) {
+    return {
+      error: {
+        errorCode: 400,
+        type: "apiFailure",
+        errorDetails: {
+          message: "API is not configured"
+        }
+      }
+    };
+  }
   const appVersion = Reaction.getAppVersion();
   const meteorVersion = _.split(Meteor.release, "@")[1];
```

Comment:
I know this isn't your code, but while we're here...

This is an unnecessary usage of `lodash` for a basic string manipulation.
```js
// nope
_.split(Meteor.release, "@")[1];

// yep
Meteor.release.split("@")[1]
```